lib/utils.js

```diff
@@ -6,6 +6,11 @@ var bind = require('./helpers/bind');
 
 var toString = Object.prototype.toString;
 
+/**
+ * Array with axios supported protocols.
+ */
+var supportedProtocols = [ 'http:', 'https:', 'file:' ];
```

Comment:
Does that `file` protocol is being tested somewhere?

Reply:
The `file` protocol is not tested in an exclusive unit test to it. I will add, thanks for tip